.spinrc.json

```diff
@@ -11,11 +11,11 @@
     },
     "android": {
       "stack": ["react-native", "android"],
-      "enabled": false
+      "enabled": true
```

Comment:
This file shouldn't be changed

Reply:
I think you commented on the previous version before I pushed the update